src/testnet.rs

```diff
@@ -1,35 +1,18 @@
 /// Parameters related to the first testnet
 pub mod one {
-    use once_cell::sync::Lazy;
-    use sunscreen::{FheRuntime, Params, PrivateKey, PublicKey, Runtime, SchemeType};
-
     use crate::fhe::FheApp;
+    use once_cell::sync::Lazy;
+    use sunscreen::{Params, SchemeType};
 
     /// Key generation and runtime parameters for the first testnet.
-    pub static PARAMS: Lazy<Params> = Lazy::new(|| Params {
+    static PARAMS: Lazy<Params> = Lazy::new(|| Params {
         lattice_dimension: 4096,
         coeff_modulus: vec![0xffffee001, 0xffffc4001, 0x1ffffe0001],
         plain_modulus: 4_096,
         scheme_type: SchemeType::Bfv,
         security_level: sunscreen::SecurityLevel::TC128,
     });
 
-    /// [`sunscreen::FheRuntime`] for the first testnet
-    pub static RUNTIME: Lazy<FheRuntime> = Lazy::new(|| Runtime::new_fhe(&PARAMS).unwrap());
```

Comment:
Can we keep this one available? Some crates don't need the full `FHE` in memory. Can we keep both runtime and params public as well?

Reply:
Sure.